### DISCUSSION TITLE
Multiple Preheat Options

### DISCUSSION THREAD

**Discussion by swissbear** [UnchartedBull/OctoDash#1285] [Ideas]
Hi
My Octodash is not yet running due to this installation script bug, but I already watched some reviews to decide which screen interface suits my needs best, and here I am ;)

When going to options and then press preheat, what about a pop up where I can choose different presets like PLA/PETG/ASA/PC/NZLCHG/etc. These presets would have to be defined somewhere too.
This would speed up things, and especially the last (nozzle change) is missing in the menu on my Prusa Mk3S+ - compiling it into the firmware is too tedious for me, and rotating that knob from 0 to 285 is always a bit tiresome.

**Comment by UnchartedBull** [comment]
There are two issues for this open now, mainly #1214. Will come some day in the future
